remme/tp/node_account.py

```diff
@@ -0,0 +1,107 @@
+import logging
+from sawtooth_sdk.processor.exceptions import InvalidTransaction
+
+from remme.protos.account_pb2 import AccountMethod
+from remme.protos.node_account_pb2 import (
+    NodeAccount, NodeAccountMethod, NodeAccountInternalTransferPayload, EmptyPayload
+)
+
+from remme.shared.forms import NodeAccountInternalTransferPayloadForm, EmptyPayloadForm
+from remme.shared.constants import Events, EMIT_EVENT
+
+from .basic import (
+    PB_CLASS, PROCESSOR, VALIDATOR, BasicHandler, get_data, get_multiple_data
+)
+from remme.settings.helper import _get_setting_value
+LOGGER = logging.getLogger(__name__)
+
+FAMILY_NAME = 'node_account'
+FAMILY_VERSIONS = ['0.1']
+
+
+def get_node_account_by_address(context, address):
```

Comment:
Use already created [get_data method](https://github.com/Remmeauth/remme-core/blob/master/remme/tp/basic.py#L69) because of it more obvious than your approach:
- your function is `tunnel function` that just wrap another one, so you make abstraction over abstraction. It makes the code harder to understand.
- your function has `side effect` — it returns a brand new object of node account if node account by address does not exist. When you `get a user`, you expect `a user` or `an error that says no user is presented`, but you do not expect a newly created user.

Reply:
Done. Function removed.